tree-sitter-go/tree-sitter-go.cabal

```diff
@@ -4,7 +4,7 @@ version:             0.2.0.1
 synopsis:            Tree-sitter grammar/parser for Go
 description:         This package provides a parser for Go suitable for use with the tree-sitter package.
 license:             BSD-3-Clause
-homepage:            https://github.com/tree-sitter/tree-sitter-go#readme
+homepage:            https://github.com/tree-sitter/haskell-tree-sitter/tree/master/tree-sitter-go
```

Comment:
I’ve changed all the `homepage` fields to point at the subdir of this repository rather than the grammar repos.

Eventually we should also have READMEs in each dir pointing at the grammar repos, and link directly to the `#readme` anchor (cf #236).